addons/stock_landed_costs/models/account_move.py

```diff
@@ -46,6 +46,11 @@ def action_view_landed_costs(self):
         views = [(self.env.ref('stock_landed_costs.view_stock_landed_cost_tree2').id, 'tree'), (False, 'form'), (False, 'kanban')]
         return dict(action, domain=domain, context=context, views=views)
 
+    def _post(self, soft=True):
+        posted = super()._post(soft)
+        self.sudo().landed_costs_ids.reconcile_landed_cost()
```

Comment:
```suggestion
        posted.sudo().landed_costs_ids.reconcile_landed_cost()
```
Also, can you protect the `sudo` in the test ? (you can simply post the bill with an accountman)